### PR TITLE
Use high res icons and remove unused

### PR DESCRIPTION
**Pull request by sithlord48**
Blocked by: #7636

Fixes: #7630 

 - Update icons  
   - Padlock 
   - Tray
   - Tray-Dark
   - Tray-Light
 - Remove unused icons
   - Warning
- Keep only  single copy of the windows icon

### CODE REVIEW

**Comment by nbolton** [comment]
Thanks for doing this. I'll test this on Monday when I get home, as I'll need to test on macOS, Windows, and Linux.

**Comment by nbolton** [comment]
Surfacing the CI error:
```
Run ./scripts/install_deps.py
CI environment detected
Using virtual environment for: /Users/runner/work/deskflow/deskflow/scripts/install_deps.py
dyld[2337]: Library not loaded: /opt/homebrew/Cellar/python@3.12/3.12.6/Frameworks/Python.framework/Versions/3.12/Python
  Referenced from: <CAFBC7FD-3E97-32A7-8EE8-EF3B8E2B346A> /Users/runner/work/deskflow/deskflow/.venv/bin/python
  Reason: tried: '/opt/homebrew/Cellar/python@3.12/3.12.6/Frameworks/Python.framework/Versions/3.12/Python' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/Cellar/python@3.12/3.12.6/Frameworks/Python.framework/Versions/3.12/Python' (no such file), '/opt/homebrew/Cellar/python@3.12/3.12.6/Frameworks/Python.framework/Versions/3.12/Python' (no such file)
Error: Process completed with exit code 250.
```

I re-ran and it still fails. Very odd. I wonder if GitHub changed their runner setup. I'll open a new PR to test this.

File: res/gui/app.qrc [hunk 1]
```diff
@@ -1,15 +1,13 @@
...
-        <file>icons/256x256/app.ico</file>
         <file>image/logo-light.png</file>
         <file>image/logo-dark.png</file>
-        <file>icons/16x16/padlock.png</file>
-        <file>icons/64x64/tray-dark.png</file>
-        <file>icons/64x64/tray-light.png</file>
+        <file>icons/64x64/padlock.png</file>
+        <file>icons/128x128/tray-dark.png</file>
+        <file>icons/128x128/tray-light.png</file>
+        <file>icons/128x128/tray.png</file>
```
Comment (nbolton) [inline res/gui/app.qrc:10]: I wonder if this should be `tray-color.png` to differentiate against light/dark?

File: res/gui/icons/128x128/tray.png
Comment (nbolton) [inline res/gui/icons/128x128/tray.png:1]: Thinking outloud: The light and dark ones are padded. I wonder if this one should have padding too? Not sure though... looks like it's used on Linux and Windows? Would this be a good opportunity to use the light and dark icons on Windows and Linux? We can detect light/dark mode now in `src/lib/gui/style_utils.h` with the `isDarkMode` function.

**Review by nbolton — APPROVED** [review]

File: res/gui/icons/128x128/tray.png
Comment (sithlord48) [reply res/gui/icons/128x128/tray.png:1]: eventually when i get to fixing the gui i will improve the iconstate.

Comment (nbolton) [reply res/gui/icons/128x128/tray.png:1]: Alright, later is fine by me.

**Pull request merged**